### PR TITLE
Revert ()

### PR DESCRIPTION
**Pull request by ICEPrey**

### CODE REVIEW

File: websites/A/Aniorb/presence.ts [hunk 1]
```diff
@@ -138,10 +136,10 @@ presence.on("UpdateData", async () => {
...
-        duration = video.duration;
-        paused = video.paused;
-        timestamps = presence.getTimestamps(
+        (currentTime = video.currentTime),
+          (duration = video.duration),
+          (paused = video.paused),
+          (timestamps = presence.getTimestamps(
             Math.floor(currentTime),
             Math.floor(duration)
           ));
```
Comment (Bas950) [inline websites/A/Aniorb/presence.ts:145]:
```suggestion
        currentTime = video.currentTime;
        duration = video.duration;
        paused = video.paused;
        timestamps = presence.getTimestamps(
            Math.floor(currentTime),
            Math.floor(duration)
          );
```

Comment (Bas950) [reply websites/A/Aniorb/presence.ts:145]: there was just an extra ) at the end of timestamp.. also upload proof of it working.

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/A/Aniorb/presence.ts (hunk 1 above)
Comment (ICEPrey) [reply websites/A/Aniorb/presence.ts:145]: do I need to upload all screenshots again?

**Review by ririxidev — APPROVED** [review]

**Comment by ririxidev** [comment]
sorry but for now im reverting your presence since it has some errors :/

**Comment by ririxidev** [comment]
more info here: https://github.com/PreMiD/Presences/runs/2401274485

**Pull request closed without merging**